data/json/items/gun/nail.json

```diff
@@ -24,5 +24,47 @@
     "reload": 50,
     "valid_mod_locations": [ [ "grip mount", 1 ], [ "rail mount", 1 ], [ "sights mount", 1 ], [ "stock mount", 1 ], [ "underbarrel mount", 1 ] ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "nail": 20 } } ]
+  },
+  {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "coilgun" },
+    "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": "2041 g",
+    "volume": "2750 ml",
+    "longest_side": "46 cm",
```

Comment:
This seems a little short short: For comparison, rifles with similar volumes:
| 2044 ml | 95 cm
| 2167 ml | 87 cm
| 2167 ml | 87 cm
| 2202 ml | 1135 mm

Reply:
Lengthened to 80 cm. The coilguns I was drawing inspiration from did appear short compared to powder-based rifles, since they don't need a long barrel to contain an explosion. Lengthening the barrel of a coilgun means adding more coils, which is probably appropriate in the context of killing zombies.